yt_dlp/extractor/radiko.py

```diff
@@ -101,33 +116,34 @@ def _find_program(self, video_id, station, cursor):
 
     def _extract_formats(self, video_id, station, is_onair, ft, cursor, auth_token, area_id, query):
         m3u8_playlist_data = self._download_xml(
-            f'https://radiko.jp/v3/station/stream/pc_html5/{station}.xml', video_id,
+            f'https://radiko.jp/v3/station/stream/aSmartPhone7a/{station}.xml', video_id,
```

Comment:
> actually yes do skip https://github.com/yt-dlp/yt-dlp/commit/e9117f0bab990ad43c1a94b3c388bc7d10437b6b
> pc_html5 api does actually return the not-live streams, its just a matter of prioritising them over the live ones

Do you mean we want to revert this?

Reply:
yes
i'll revert that commit and then apply your other fixes
~~cant be bothered to~~ not sure how id go about prioritising the not-live so i'll just leave it for now